src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1864,6 +1868,44 @@ export default class ComparisonStore {
         {}
     );
 
+    readonly survivalXAxisLabelGroupByPrefix = remoteData({
+        await: () => [
+            this.activeStudiesClinicalAttributes,
+            this.survivalClinicalAttributesPrefix,
+        ],
+        invoke: () => {
+            const survivalXAxisLabelGroupByPrefix = _.reduce(
+                this.survivalClinicalAttributesPrefix.result!,
+                (acc, prefix) => {
+                    const clinicalAttributeId = `${prefix}_MONTHS`;
+                    const clinicalAttributeMap = _.groupBy(
+                        this.activeStudiesClinicalAttributes.result,
+                        'clinicalAttributeId'
```

Comment:
don't use string shortcuts b/c they aren't typesafe
(n)=>n.clinicalAttributeId

Reply:
also, i don't think building this map is necessary.  just filter them by clinicalAttributeId=match.